BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/BTConnector.java

```diff
@@ -46,45 +43,38 @@ public WifiBtStatus(){
     }
 
     public interface  Callback{
```

Comment:
The name "Callback" is more than a little generic and makes it had to follow the code when it is used outside of this class, such as in BtConnectorHelper. How about a more descriptive name like BTConnectorCallback?